content/articles/spring-boot-caching/index.md

```diff
@@ -0,0 +1,332 @@
+Cache reduces the number of database reads resulting in reduced cost of operation and improved application speed. Spring Boot provides an abstraction layer for cache implementation. 
+		
+
+In this article, we will learn how to implement a cache in a Spring Boot REST application using Ehcache as the cache provider.
```

Comment:
What's Ehcache? Add any reference links for it.

Why are we using it? What is a cache provider?